Automattic/jetpack
# Carousel: Add a version number to swiper global to prevent clashes with 3rd party plugin uses of swiper

#### Changes proposed in this Pull Request:

* Adds a version number to the end of the window.Swiper global to prevent an clashes with other plugins/themes that may be loading swiper

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Check out PR to local dev env
* Enable the carousel option in Jetpack settings
* Add a gallery and view in front end and make sure carousel works as expected
